api/v1/installation_types.go

```diff
@@ -421,13 +421,16 @@ const (
 
 // LinuxDataplaneOption controls which dataplane is to be used on Linux nodes.
 //
-// One of: Iptables, BPF
+// One of: Iptables, BPF, VPP, Nftables, BPFWithNftables
+// +kubebuilder:validation:Enum=Iptables;BPF;VPP;Nftables;BPFWithNftables
 type LinuxDataplaneOption string
 
 const (
 	LinuxDataplaneIptables LinuxDataplaneOption = "Iptables"
 	LinuxDataplaneBPF      LinuxDataplaneOption = "BPF"
 	LinuxDataplaneVPP      LinuxDataplaneOption = "VPP"
+	LinuxDataplaneNftables LinuxDataplaneOption = "Nftables"
+	LinuxDataplaneBPFNFT   LinuxDataplaneOption = "BPFWithNftables"
```

Comment:
Submitting for discussion. Not sure the best way to represent this - we can have nftables mode, but also BPF mode with nftables instead of iptables

Reply:
I guess an alternative would be to add a new field that is only applicable in BPF dataplane mode that is either Nftables or Iptables (default).

Reply:
Yeah, that's basically the only alternative I thought of. But also feels like an awkward UX.

e.g.,

```go
linuxDataplane: BPF
bpfTablesMode: Nftables
```

I think perhaps for now I stick with just `Iptables`, `BPF`, `VPP`, `Nftables` in order to get this merged, and consider BPF + Nftables separately (it's not even 100% working yet anyway).